### PR TITLE
Update protobuf definitions to v1.12.2

### PR DESCRIPTION
**Pull request by gregeinfrank**
Updates to latest Envoy API [v1.12.2] (as of Dec 10, 2019)

Followed the instructions: https://github.com/envoyproxy/java-control-plane#bring-api-up-to-date-with-data-plane-api

### CODE REVIEW

**Comment by gregeinfrank** [comment]
@snowp - Please let me know if there's anything else needed here

File: tools/API_SHAS [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-GOGOPROTO_SHA="4cbf7e384e768b4e01799441fdf2a706a5635ae7" # v1.2.0
-GOOGLEAPIS_SHA="be480e391cc88a75cf2a81960ef79c80d5012068"  # Jul 24, 2019
-OPENCENSUS_SHA="a105b96453fe85139acc07b68de48f2cbdd71249" # Mar 13, 2018 (tag v0.2.0)
-PGV_GIT_SHA="4349a359d42fdfee53b85dd5c89a2f169e1dc6b2" # Jan 30, 2019  (tag v0.0.13)
+# dependencies (source: https://github.com/envoyproxy/envoy/blob/3c899ffab8092ae7adf34042644ea14cb2404e07/api/bazel/repository_locations.bzl)
+GOOGLEAPIS_SHA="82944da21578a53b74e547774cf62ed31a05b841"  # Dec 2, 2019
+OPENCENSUS_SHA="5cec5ea58c3efa81fa808f2bd38ce182da9ee731" # July 25, 2019
+PGV_GIT_SHA="a18376249eb51cdd517f67fe8703897322812e6d" # Nov 5, 2019
 PROMETHEUS_SHA="99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c" # Nov 17, 2017
+UDPA_SHA="d1f2ba7f5ba62c55b7466409e7f972c93e957d2b"  # Dec 6, 2019
```
Comment (gregeinfrank) [inline tools/API_SHAS]: Note the changes here, `GOGOPROTO` was removed and `UDPA` was added

File: tools/update-api.sh [hunk 2]
```diff
@@ -56,4 +52,8 @@ cp -r opencensus-proto-*/src/opencensus/proto/* "${protodir}/opencensus/proto"
 curl -sL https://github.com/prometheus/client_model/archive/${PROMETHEUS_SHA}.tar.gz | tar xz --include="*.proto"
 cp client_model-*/metrics.proto "${protodir}"
 
+curl -sL https://github.com/cncf/udpa/archive/${UDPA_SHA}.tar.gz | tar xz --include="*.proto"
```
Comment (gregeinfrank) [inline tools/update-api.sh]: And here as well

**Comment by jakubdyszkiewicz** [comment]
LGTM @gregeinfrank merging to master. Thank you.

**Comment by gregeinfrank** [comment]
Thanks @jakubdyszkiewicz  - can you push a new release for v1.12.2?

**Comment by gregeinfrank** [comment]
@jakubdyszkiewicz ping -- any idea when we can push a new version?

**Comment by jakubdyszkiewicz** [comment]
Sorry about the delay. @chemicL @slonka how do you feel about releasing a new version?
Can we do it or do you want to also push changes in pending PRs?

**Comment by slonka** [comment]
I think we can release this, we probably want to release our changes in a separate version.

**Comment by jakubdyszkiewicz** [comment]
Hey, @gregeinfrank I released java control plane 0.1.21.
I'll try to be more responsive next time, but also feel free to ping me on Envoy Slack anytime.

**Comment by gregeinfrank** [comment]
@jakubdyszkiewicz Thanks. Any idea how to get this up on https://jcenter.bintray.com/com/github/envoy-scala/control-plane/

**Comment by jakubdyszkiewicz** [comment]
I think this is completely different project (envoy-scala?). This one is available in maven central here
https://search.maven.org/artifact/io.envoyproxy.controlplane/java-control-plane/0.1.21/pom

**Pull request merged**